runtimes/runtimes/standalone.ts

```diff
@@ -166,20 +166,17 @@ export const standalone = (props: RuntimeProps) => {
                 }
             },
             fs: {
-                copy: (src, dest) => {
+                copyFileWithCreateDir: (src, dest) => {
```

Comment:
That's a bit weird name. Should be leave `copyFile` as is in NodeJs and let servers create the folder? Or expose `ensureDir` as an option?

Reply:
I think we have the freedom to stay mostly consistent with nodejs while providing some nice features as a built in. This might be a radical suggestion but I think we can keep this function as is 

I feel like exposing ensureDir at this point is redundant

Reply:
Having `ensureDir` in options is more flexible to support future possible cases. Also, it would give possibility to leave the previous name, which is way better.

Reply:
I'm inclined towards adding `ensureDir` option, because it describes what function does internally.